.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-T2C
+--strict-checksums
```

Comment:
`--strict-checksums` made it in, but this is a good thing: Maven will reject dependencies with checksum failures. Surprisingly, this is not the default.